kernel/classes/ezscript.php

```diff
@@ -235,8 +235,17 @@ function startup()
     */
     function initialize()
     {
-        if( ob_get_length() != 0 )
-            ob_end_clean();
+        try
+        {
+            ezpKernel::instance();
```

Comment:
OK I get it. But this is really hackish...
Actually we should better have an instance of `ezpKernel` ready in CLI, like in platform stack